CONTRIBUTING.md

```diff
@@ -12,6 +12,12 @@ The project README file includes all the information you need to build a local v
 
 We use the [Airbnb style guide](https://github.com/airbnb/javascript), except with semicolons turned off. We suggest using Prettier in your environment to make sure your code files are all balanced and beautiful.
 
+### Style and SCSS Modules
+
+We use Gatsby's [CSS modules](https://www.gatsbyjs.org/docs/css-modules/), but with Sass files. Every Sass file in `/src/scss` should be in a file that mirrors the structure of their parent component. For example, a component at `/src/components/beetle.js` that needs styling should have a corresponding SCSS file at `/src/scss/components/beetle.module.scss`.
```

Comment:
Thinking through this a bit: maybe it’d make sense to consolidate styles alongside their components? So rather than

- `/src/components/beetle.js`
- `/src/scss/components/beetle.module.scss`

maybe something like


- `/src/components/beetle/index.js`
- `/src/components/beetle/style.module.scss`

?

(Don’t feel strongly about this, and this is not a blocker for merge; just curious!)

Reply:
I'm already ripping the site's SCSS files apart, so I'll change them all to adjacent files very soon.